packages/asset-swapper/src/utils/quote_report_generator.ts

```diff
@@ -116,6 +137,85 @@ export function generateQuoteReport(
     };
 }
 
+/**
+ * Generates a report of sources considered while computing the optimized swap quote, the sources ultimately included in the computed quote, and all considered quotes.
+ */
+export function generateExtendedQuoteReport(
```

Comment:
now that we have `generateExtendedQuoteReport`, is it possible for us to remove `generateQuoteReport`?

Reply:
Not yet, we want a period where both work in parallel to compare across versions